src/FileSystem/FilesFinder.php

```diff
@@ -39,6 +39,7 @@ public function findInDirectoriesAndFiles(array $source, array $suffixes = [], b
 
         // filtering files in files collection
         $filteredFilePaths = $this->fileAndDirectoryFilter->filterFiles($filesAndDirectories);
+        $filteredFilePaths = array_map($this->filePathHelper->absolutePath(...), $filteredFilePaths);
```

Comment:
just use `realpath()` should be enough:

```php
        $filteredFilePaths = array_map(realpath(...), $filteredFilePaths);
```

also, place after last `$filteredFilePaths` after line 69 so less IO used

Reply:
Ok, using here seems fine, as on directory filter, the path is got realpath first before next PathSkipper usage

https://github.com/rectorphp/rector-src/blob/a4545b6fccbc6698fadce47a8454ce3acebf97ac/src/FileSystem/FilesFinder.php#L132-L149